src/main/java/com/twcable/grabbit/tools/jobstarter/HostAndJobIds.java

```diff
@@ -0,0 +1,60 @@
+/*
+ * Copyright 2014-2016 Time Warner Cable, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.twcable.grabbit.tools.jobstarter;
+
+import org.checkerframework.checker.nullness.qual.Nullable;
+import org.reactivestreams.Publisher;
+
+import java.net.URI;
+
+/**
+ * Tuple of a host URI and a Publisher of job ids.
+ */
+@SuppressWarnings({"WeakerAccess", "checkstyle:VisibilityModifier", "checkstyle:NeedBraces"})
+public class HostAndJobIds {
+    public final URI uri;
```

Comment:
I did a lot of `public final ...` fields. The primary problem with making a field public is your don't want it mutating, but the `final` takes care of that. In theory, having a getter also gives the ability to compute the value if for some reason that becomes useful ("referential transparency" for cache etc.) but I don't see that as being a practical issue for this project. Besides, the amount of boilerplate already needed because of `equals()` and `hashCode()` and `toString()` and constructors is already pretty high, so adding getters would just add more noise.